alexBot/cogs/video_dl.py

```diff
@@ -87,17 +88,23 @@ async def on_message(self, message: discord.Message):
             return
 
         matches = None
-
-        for regex in REGEXES:
+        convert_reddit = False
+        
+        for i,regex in enumerate(REGEXES):
             matches = regex.match(message.content)
             if matches:
+                convert_reddit = i == 3
```

Comment:
This approach (hardcorded index) isn't incredibly maintainable. A better approach is to have a separate list of regexs and check if in the seperate regex list